lib/host.rb

```diff
@@ -854,20 +854,36 @@ def exec_raw(*commands, **opts)
         commands = ["chroot", "/host/", "bash", "-c", commands_to_string(commands)]
       end
 
+      unless node.env.opts[:host_debug_image] == "support-tools"
+        @result = node.env.admin.cli_exec(:get, n:"openshift-multus", resource: "pod", l: "app=multus", output: 'jsonpath={.items[0].spec.containers[?(@.name=="kube-multus")].image}')
+        if @result[:success]
+          # we could cache this but then we would have to re-get after upgrade
+          opts[:image] = @result[:response].strip
+        end
```

Comment:
Maybe better use our object model here. Something like
```
mpods = Pod.get_labeled("app=multus", user: node.env.admin, project: Project.new(name: "openshift-multus", env: node.env))
opts[:image] = mpods.first.container(name: "kube-multus").spec.image.name
```

But not sure which one is more readable for others. wdyt?

Reply:
The opt name should be `debug_image` or something like that though. `image` sounds too generic for the `Host` object.

Reply:
I tried using `Pod.get_labeled` but couldn't figure out the `Project` or `user` setting and gave up.  I'll try again.

will also try `opts[:debug_image]`

Reply:
err I guess we don't need `opts[:debug_image]` that was from an old patch where I wanted to set the debug image using a optional step arg passed all the way down to `Host.exec_raw`

I'll remove the use of `opts`.